pkg/config/options.go

```diff
@@ -68,6 +68,8 @@ type KanikoOptions struct {
 	ImageNameDigestFile      string
 	ImageNameTagDigestFile   string
 	OCILayoutPath            string
+	Compression              string
```

Comment:
Could this be a `CompressionLevel` type like you have in the go-containerregistry change?

Reply:
If we expose the `Compression` type in the go-containerregistry package I'll just use this type. Otherwise I'll create a new one in this codebase.